packages/uops-dashboard/src/server/clients/apiUrls.ts

```diff
@@ -43,7 +45,7 @@ export function getScanUrl(chainId: ChainId): string {
 }
 
 export function getApiKey(chainId: ChainId, type: ApiKeyType): string {
-  const envName = `${chainId.toUpperCase()}_${type}_API_KEY`
+  const envName = `${chainId.toUpperCase().replace('-', '_')}_${type}_API_KEY`
```

Comment:
why is there a need for this change?

Reply:
the name of the var is `ZKSYNC_ERA_SCAN_API_KEY` so `zksync-era` get's translated to `ZKSYNC_ERA`